entity-framework/core/providers/sql-server/functions.md

```diff
@@ -9,6 +9,23 @@ uid: core/providers/sql-server/functions
 
 This page shows which .NET members are translated into which SQL functions when using the SQL Server provider.
 
+## Aggregate functions
+
+.NET                                                                    | SQL                              | Added in
+----------------------------------------------------------------------- | -------------------------------- | --------
+EF.Functions.StandardDeviationSample(group.Select(x => x.Property))     | STDEV(Property)                  | EF Core 7.0
```

Comment:
We should probably have a general conceptual doc section showing how to use aggregate functions in various scenarios, opened #4002.

BTW technically the thing that GroupBy returns is called a "grouping" rather than a group, we may want to align (though I think group is OK too).

For reference, here are the [PG translations](https://www.npgsql.org/efcore/mapping/translations.html#aggregate-functions).

Reply:
> technically the thing that GroupBy returns is called a "grouping" rather than a group

lol, I had the same internal debate. In the end, I went for the shorter one.